src/CabanaPD_HeatTransfer.hpp

```diff
@@ -0,0 +1,123 @@
+/****************************************************************************
+ * Copyright (c) 2022-2023 by Oak Ridge National Laboratory                 *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of CabanaPD. CabanaPD is distributed under a           *
+ * BSD 3-clause license. For the licensing terms see the LICENSE file in    *
+ * the top-level directory.                                                 *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef HEATTRANSFER_H
+#define HEATTRANSFER_H
+
+#include <CabanaPD_Timer.hpp>
+
+namespace CabanaPD
+{
+
+// Peridynamic heat transfer with forward-Euler time integration.
+// Inherits only because this is a similar neighbor-based kernel.
+template <class ExecutionSpace, class ModelType>
+class HeatTransfer : public Force<ExecutionSpace, BaseForceModel>
```

Comment:
Should be specialized on `Elastic`

Reply:
What does this comment mean?

Reply:
Just that the next PR should address a separate version with fracture

Reply:
Sure. However, is there a need to make any changes here based on your comment?

Reply:
OK